app/src/main/java/de/westnordost/streetcomplete/overlays/cycleway/StreetCyclewayOverlayForm.kt

```diff
@@ -193,18 +226,12 @@ class StreetCyclewayOverlayForm : AStreetSideSelectOverlayForm<CyclewayAndDirect
     }
 
     override fun onClickOk() {
-        if (bicycleBoulevard == BicycleBoulevard.YES) {
-            val tags = StringMapChangesBuilder(element!!.tags)
-            bicycleBoulevard.applyTo(tags, countryInfo.countryCode)
-            applyEdit(UpdateElementTagsAction(element!!, tags.create()))
+        // only tag the cycleway if that is what is currently displayed
```

Comment:
Does this comment still make sense? I think now everything that changed is always tagged.

Reply:
true